Assets/MixedRealityToolkit/_Core/Resources/Textures/Oculus lineart attribution.txt

```diff
@@ -0,0 +1,5 @@
+You may use these images solely for referring to the corresponding product in your video game or VR experience (including manuals for users). Otherwise, you may not use these images, or any Oculus trademarks, logos or other intellectual property, including but not limited to use on merchandise or other product such as clothing, hats, or mugs. Do not use the Oculus images in a way that implies a partnership, sponsorship or endorsement; or features Oculus on materials associated with pornography, illegal activities, or other materials that violate Oculus Terms.
```

Comment:
Would the MRTK be correctly categorized under "video game or VR experience"? I'm not sure.

Reply:
I would say so, yes.